tests/test_io.py

```diff
@@ -253,6 +253,32 @@ def test_write_macro_block_size():
     assert frame_sizes[1] == (40, 50)
 
 
+def test_write_big_frames():
+    """Test that we give ffmpeg enough time to finish."""
+    try:
+        import numpy as np
+    except ImportError:
+        return skip("Missing 'numpy' test dependency")
+
+    sizes = []
+    for pixfmt, bpp in [("gray", 1), ("rgb24", 3), ("rgba", 4)]:
+        # Prepare for writing
+        gen = imageio_ffmpeg.write_frames(
+            test_file2, (2048, 2048), pix_fmt_in=pixfmt, ffmpeg_timeout=20.0
+        )
+        gen.send(None)  # seed
+        for i in range(9):
+            data = (255 * np.random.rand(2048 * 2048 * bpp)).astype(np.uint8)
+            data = bytes(data)
+            gen.send(data)
+        gen.close()
+        with open(test_file2, "rb") as f:
+            sizes.append(len(f.read()))
```

Comment:
Is seems like sizes is not used.

Reply:
This test was mostly copied from other tests by @jmetz and existing tests like `test_write_pix_fmt_in` create `sizes` but don't use it. Would you like it removed or a check added?

Reply:
Ya sorry - was just quick copy-edit job to create a test that caused the process killing issue. Hadn't fully audited the code.

Reply:
Doing the size check shows some odd results, at least from what I'd expect. The 'rgb24' file is larger than the 'rgba' file. Maybe has to do with it removing certain alpha=0 pixels?

Reply:
Maybe ffmpeg uses different  compression ratios for rgb24 and rgba?